packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -236,6 +286,16 @@ class TextPainter {
 
   ui.Paragraph _layoutTemplate;
 
+  List<TextBox> _inlinePlaceholderBoxes;
+  List<TextBox> get inlinePlaceholderBoxes => _inlinePlaceholderBoxes;
+
+  List<PlaceholderDimensions> _placeholderDimensions;
+  set placeholderDimensions(List<PlaceholderDimensions> value) {
```

Comment:
setter without getter should either be a method or should have a getter

also, missing dartdocs on public member